core/chain-configs/src/genesis_config.rs

```diff
@@ -0,0 +1,125 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+
+use near_primitives::serialize::u128_dec_format;
+use near_primitives::state_record::StateRecord;
+use near_primitives::types::{
+    AccountId, AccountInfo, Balance, BlockHeightDelta, Gas, NumBlocks, NumSeats,
+};
+use near_runtime_configs::RuntimeConfig;
+
+use crate::PROTOCOL_VERSION;
+
+pub const CONFIG_VERSION: u32 = 1;
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct GenesisConfig {
```

Comment:
Leave a comment saying that `chain-configs` is not the best place for `GenesisConfig` since it contains `RuntimeConfig`, but we keep it here for now until we figure out the better place.

Reply:
Done